package.json

```diff
@@ -2,47 +2,52 @@
   "name": "@mswjs/interceptors",
   "description": "Low-level HTTP/HTTPS/XHR/fetch request interception library.",
   "version": "0.19.3",
-  "main": "./lib/node/index.mjs",
+  "main": "./lib/node/index.js",
+  "module": "./lib/node/index.mjs",
   "types": "./lib/node/index.d.ts",
   "exports": {
     ".": {
       "types": "./lib/node/index.d.ts",
-      "node": "./lib/node/index.js",
+      "default": "./lib/node/index.js",
       "require": "./lib/node/index.js",
-      "default": "./lib/node/index.mjs"
+      "import": "./lib/node/index.mjs"
     },
     "./ClientRequest": {
       "types": "./lib/node/interceptors/ClientRequest/index.d.ts",
-      "node": "./lib/node/interceptors/ClientRequest/index.js",
+      "default": "./lib/node/interceptors/ClientRequest/index.js",
       "require": "./lib/node/interceptors/ClientRequest/index.js",
-      "default": "./lib/node/interceptors/ClientRequest/index.mjs",
+      "import": "./lib/node/interceptors/ClientRequest/index.mjs",
       "browser": null
     },
     "./XMLHttpRequest": {
       "types": "./lib/node/interceptors/XMLHttpRequest/index.d.ts",
-      "node": "./lib/node/interceptors/XMLHttpRequest/index.js",
+      "default": "./lib/node/interceptors/XMLHttpRequest/index.js",
       "require": "./lib/node/interceptors/XMLHttpRequest/index.js",
+      "import": "./lib/node/interceptors/XMLHttpRequest/index.mjs",
       "browser": {
         "types": "./lib/browser/interceptors/XMLHttpRequest/index.d.ts",
         "require": "./lib/browser/interceptors/XMLHttpRequest/index.js",
-        "default": "./lib/browser/interceptors/XMLHttpRequest/index.mjs"
+        "default": "./lib/browser/interceptors/XMLHttpRequest/index.js",
+        "import": "./lib/browser/interceptors/XMLHttpRequest/index.mjs"
       }
     },
     "./fetch": {
       "types": "./lib/node/interceptors/fetch/index.d.ts",
-      "node": "./lib/node/interceptors/fetch/index.js",
+      "default": "./lib/node/interceptors/fetch/index.js",
       "require": "./lib/node/interceptors/fetch/index.js",
+      "import": "./lib/node/interceptors/fetch/index.mjs",
       "browser": {
         "types": "./lib/browser/interceptors/fetch/index.d.ts",
         "require": "./lib/browser/interceptors/fetch/index.js",
-        "default": "./lib/browser/interceptors/fetch/index.mjs"
+        "default": "./lib/browser/interceptors/fetch/index.js",
+        "import": "./lib/browser/interceptors/fetch/index.mjs"
       }
     },
     "./RemoteHttpInterceptor": {
       "types": "./lib/node/interceptors/RemoteHttpInterceptor.d.ts",
-      "node": "./lib/node/interceptors/RemoteHttpInterceptor.js",
+      "default": "./lib/node/interceptors/RemoteHttpInterceptor.js",
```

Comment:
I think default has to come last in the object

These are resolved in the order they're listed in the object, so we might need 

```
browser
require
default (or import)
```